mathesar/models.py

```diff
@@ -112,6 +112,10 @@ def sa_columns(self):
     def sa_column_names(self):
         return self.sa_columns.keys()
 
+    @property
+    def has_dependencies(self):
```

Comment:
@eito-fis Could you update this to actually check if it has dependencies? If that's going to be a lot of work, we can separate it into a new issue, but we should at least add a `TODO` here that indicates it's a placeholder and needs to be replaced.